sdk/formrecognizer/ai-form-recognizer/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ### Breaking Changes
 
+- [#20775](https://github.com/Azure/azure-sdk-for-js/pull/20775) Renames in `DocumentModelAdministrationClient`,
+
+  - Method `beginCopyModel()` -> `beginCopyModelTo()`
+  - In the method `beginComposeModel()`, renamed argument `componentModels` -> `componentModelIds`.
```

Comment:
The rest of the changelog is written in formal prose.

I would replace this with ``- Renamed the `beginCopyModel` method of `DocumentModelAdministrationClient` to `beginCopyModelTo`.``. And you can link the PR at the end if you like.

The second bullet point here is not a breaking change and shouldn't be included in this section.

Reply:
```suggestion
- Renamed the `beginCopyModel` method of `DocumentModelAdministrationClient` to `beginCopyModelTo`. [#20775](https://github.com/Azure/azure-sdk-for-js/pull/20775) 
```